zerver/models.py

```diff
@@ -1982,7 +1984,7 @@ class CustomProfileField(models.Model):
         (CHOICE, str(_('Choice')), validate_choice_field, str),
     ]  # type: FieldTypeData
     USER_FIELD_TYPE_DATA = [
-        (USER, str(_('User')), check_valid_user_id, int),
+        (USER, str(_('User')), check_valid_user_ids, eval),
```

Comment:
It's almost never a good idea to use `eval` with anything involving user input; why do you have this?

Reply:
It's used to convert string representation of list into python list for user profile data.